src/movies.js

```diff
@@ -1,25 +1,161 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+ 
+    const newArray = moviesArray.map( (element) => element.director)
+    return newArray;
+
+}
+
```

Comment:
Una vueltita mas que es lo mismo, si lo que haces lo metes en una variable y luego devuelves esa variable sin modificarla, devuelve lo que contiene la variable, es decir

```suggestion
function getAllDirectors(moviesArray) {
    return moviesArray.map( (element) => element.director)
}
```